src/api/authorizeApi.ts

```diff
@@ -16,8 +17,12 @@ export default class AuthorizeApi {
         }
       })
       .catch((error) => {
-        if (error.response.status === 400) {
-          notificationLogic("error", error.response.data.value);
+        if (error.response.data.value == "User-FormerMember") {
+          showUserIsFormerMemberModal();
+        } else {
```

Comment:
For what reason you need here this `else` statement?

Reply:
Because without it there will be additional unnecessary notification

Reply:
Done!